survey-sdk/src/main/java/co/nimblehq/survey/sdk/entity/MetaEntity.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.survey.sdk.entity
+
+import com.squareup.moshi.Json
+
+/*
+This class not be included into json-api.
+Hence, we use @Json annotation for Parsing with Moshi Adapter
+ */
```

Comment:
With the multiple lines of comments. We should add like this 😄 

```
/*
 * Block comment
 * that is longer than 2 lines
 */
```

Reference: https://nimblehq.co/compass/development/code-conventions/kotlin/#comments

Reply:
@hoangnguyen92dn good to know ❤️